overview.md

```diff
@@ -11,13 +11,15 @@ However, as this idea evolved, we ran into a big problem: we don't know who
 deserves credit in an open-source project, and thus we don't know how to
 distribute the rewards. The obvious approaches, like counting commits or lines
 of code, would be far too easy to game—and would in any case miss out on vital
-work like design or issue triaging.
+work like design or issue triaging, not to mention that it would miss out on the
+underlying quality of the commits or code.
```

Comment:
Even before the modification, this sentence is pretty ungainly:
The obvious approaches (descriptive subject)
like counting commits or lines of code (examples provided)
would be easy to game (passive tense property of the subject)
-and would in any case miss out on vital work (another property of the subject)
like design or issue triaging (examles of another property)

so I think adding in another clause with more examples makes it kind of unmanageable. I think pointing out that the naive approaches are deficient w.r.t picking up the right kinds of work *and* discerning the quality of work is a valuable one, but I don't want to overload this sentence. could you re-write to factor these concepts apart more clearly?